tests/auth_tests/test_forms.py

```diff
@@ -191,9 +204,9 @@ def test_duplicate_normalized_unicode(self):
     )
     def test_validates_password(self):
         data = {
-            "username": "testclient",
-            "password1": "testclient",
-            "password2": "testclient",
+            "username": "testclient2",
+            "password1": "testclient2",
+            "password2": "testclient2",
```

Comment:
This change looks unrelated :thinking: Please revert.

Reply:
I reverted the changes and renamed UserCreationFormTest to BaseUserCreationFormTest. The tests under the latter are now identical to the tests of the old UserCreationFormTest